spec/features/merchant/items_spec.rb

```diff
@@ -116,6 +116,28 @@
       expect(page).to have_content("Skinny Tires")
       expect(page).to have_content("Half the tire at twice the cost :^)")
     end
+
+    it "wont let me edit items to have attributes that arnt valid" do
```

Comment:
Just a quick thing to change this to aren't

Reply:
Florida